src/packages/util/db-schema/name-rules.ts

```diff
@@ -112,7 +112,7 @@ export function checkPublicPathName(name: string) {
rating milestones mine mirrors mis misc mob mobile module modules monitor motherfucker movie movies mp3 msg msn music musicas mx my mysql name named nan navi navigation net network new news newsletter nick nickname node nodes none nonprofit nonprofits notes notices noticias notification notifications notify ns ns1 ns10 ns2 ns3 ns4 ns5 ns6 ns7 ns8 ns9 null oauth oauth_clients offer offers office official old online open-source openid operator order orders organisations organization organizations orgs overview owner owners page pager pages panel partners password password-reset payment payments perl personal phone photo photoalbum photos php phpmyadmin phppgadmin phpredisadmin pic pics ping piss plan plans plugin plugins policies policy poll polls pop pop3 popular popularity portal post postfix postmaster posts pr premium press price pricing primus privacy privacy-policy privacy_policy privacypolicy private processes product products professional profile profiles project projects promo pub public pulls purpose put python query random ranking raw rdf rdfs read readme recent recommendations recruit recruitment redeem register registration release releases remove render replies reply report reports repositories repository req request requests reset resources restore revert roc root rss ruby rule sag sagemath sagemathinc sale sales sample samples save save-net-neutrality saved school scraping script scripts search secure security self send server server-info server-status service services session sessions setting settings setup share shareholders shit shop show showcases sign-in sign-up sign_in sign_up signin signout signup site sitemap sites smartphone smtp snapshots software soporte source spam spec special sponsors sql src ssh ssl ssladmin ssladministrator sslwebmaster staff stage staging star starred stars start stat state static statistics stats status statuses storage store stores stories style styleguide stylesheet stylesheets subdomain subscribe subscriptions suggest suggestion suggestions suporte support survey surveys suspended svn swf sys sysadmin sysadministrator system tablet tablets tag talk talks task tasks teach teacher teachers teaching team teams tech telnet template templates ten term terms terms-of-service terms_of_service termsofservice test test1 test2 test3 teste testing tests theme themes thread threads timeline timetravel tits tmp todo tool tools top topic topics tos tour train training translations tree trending trends try tutorial tutorials tux tv twitter twittr undef unfollow unsubscribe update updates upload uploads url usage user username users usuario vendas ver version video videos visitor visualization w watch watching weather web webapp webhook webhooks webmail webmaster website websites welcome widget widgets wiki win windows word work works works-with workshop ww wws www www0 www1 www2 www3 www4 www5 www6 www7 www8 www9 wwws wwww xfn xml xmpp xpg xxx yaml year yml you yourdomain yourname yoursite yourusername`.split(
```

Comment:
This makes no sense, because the name rules don't allow any of ".well-known robots.txt humans.txt", since no dots are allowed.  Copying from the comment above:

Account name:
 - between 1 and 39 characters
 - doesn't start with a -
 - only includes the characters 0-9,a-z,A-Z,-
 - Don't allow uuid's.
 - cannot include consecutive hyphens